src/JoomlaBrowser.php

```diff
@@ -137,6 +137,7 @@ public function doAdministratorLogin($user = null, $password = null, $useSnapsho
 		$this->fillField($this->locator->adminLoginUserName, $user);
 		$this->debug('Fill Password Text Field');
 		$this->fillField($this->locator->adminLoginPassword, $password);
+		$this->makeScreenshot('loginPage');
```

Comment:
The branch you made this on is `v4.0.0` are you sure or you want it for the new template?

Reply:
I want it for 4.0-dev. The failure to login is currently failing around 1/4 of tests in drone. It’s the last major random failure that didn’t get fixed in the ATT sprint

Reply:
I was lazy 😂